src/swap/letsexchange.js

```diff
@@ -0,0 +1,268 @@
+// @flow
+
+import { lt } from 'biggystring'
+import { asObject, asString } from 'cleaners'
+import {
+  type EdgeCorePluginOptions,
+  type EdgeCurrencyWallet,
+  type EdgeSpendInfo,
+  type EdgeSwapInfo,
+  type EdgeSwapPlugin,
+  type EdgeSwapQuote,
+  type EdgeSwapRequest,
+  type EdgeTransaction,
+  SwapBelowLimitError,
+  SwapCurrencyError
+} from 'edge-core-js/types'
+
+import {
+  type InvalidCurrencyCodes,
+  checkInvalidCodes,
+  getCodesWithMainnetTranscription,
+  makeSwapPluginQuote
+} from '../swap-helpers.js'
+import { asOptionalBlank } from './changenow'
+
+const pluginId = 'letsexchange'
+const swapInfo: EdgeSwapInfo = {
+  pluginId,
+  displayName: 'LetsExchange',
+  supportEmail: 'support@letsexchange.io'
+}
+
+const orderUri = 'https://letsexchange.io/?exchangeId='
+const uri = 'https://api.letsexchange.io/api/v1/'
+
+const expirationMs = 1000 * 60 * 20
+
+const asQuoteInfo = asObject({
+  transaction_id: asString,
+  status: asString,
+  coin_from: asString,
+  coin_to: asString,
+  coin_from_network: asString,
+  coin_to_network: asString,
+  deposit_amount: asString,
+  withdrawal_amount: asString,
+  deposit: asString,
+  deposit_extra_id: asOptionalBlank(asString),
+  withdrawal: asString,
+  withdrawal_extra_id: asOptionalBlank(asString),
+  rate: asString,
+  fee: asString,
+  return: asString,
+  hash_in: asOptionalBlank(asString),
+  hash_out: asOptionalBlank(asString)
+})
+
+const dontUseLegacy = {
+  DGB: true
+}
+
+const INVALID_CURRENCY_CODES: InvalidCurrencyCodes = {
+  from: {
+    ETH: ['MATIC'],
```

Comment:
It appears the LetsExchange API supports identifying the mainnet currency when creating orders so we can loosen up the limits here. Another plugin that can similarly use mainnet codes is Changenow you can likely use a similar list of invalid codes. 

A note about ZEC, Edge only creates shielded addresses so if LetsExchange cannot send to a shielded address than it should remain invalid here.

Reply:
Done!